browser/views.py

```diff
@@ -657,22 +656,46 @@ def add_investigator_record(request, person_id):
     template = "browser/investigator.html"
     form = InvestigatorForm()
 
+    context = {
+        'form': form,
+        'person': person,
+        'type': 'create_investigator'
+    }
```

Comment:
the parameters that are not needed in the post request, should be added to the context before it returns (when they are actually needed).

Reply:
@jdamerow Is <b>type</b> parameter the one which you are pointing to ?